cypress/integration/cohorts/index.spec.js

```diff
@@ -0,0 +1,118 @@
+
+describe('Cohorts Screen', () => {
+  
+  beforeEach(() => {
+    /*
+      Cypress starts out with a blank slate for each test
+      so we must tell it to visit our website with the `cy.visit()` command.
+      Since we want to visit the same URL at the start of all our tests,
+      we include it in our beforeEach function so that it runs before each test
+    */
+    
+    cy.handleLogin()
+    
+  });
+  context('Testing cohorts Screen', () => {
+     
+    it('Checking that the button creates a new cohort', () => {
+        
+      cy.fixture('/cohorts_screen_values/values').then((values) => {
+        cy.log('**_____ Sending the mocked cohorts_____**')
+        cy.mock_list_cohortsA()
+        
+        cy.log('**_____ Goin to Cohorts Screen... _____**')
+        cy.visit('/admissions/cohorts')
+        cy.wait('@mock_list_cohortsA')
+        cy.wait(1500)
+
+        cy.log('**_____ Sending the syllabus certificates names_____**')
+        cy.syllabus_certificates()
+
+        cy.log('**_____ Sending the certificates versions_____**')
+        cy.certificates_versions()
+
+        cy.log('**_____ Creating a new cohort _____**')
+        cy.get('.btn > .MuiButtonBase-root').click()
+
+        cy.log('**_____ Entering the new cohort NAME _____**')
+        cy.get(':nth-child(2) > .MuiFormControl-root > .MuiInputBase-root > .MuiInputBase-input')
+        .type(values.name)
+
+        cy.log('**_____ Entering the new cohort SLUG _____**')
+        cy.get(':nth-child(4) > .MuiFormControl-root > .MuiInputBase-root > .MuiInputBase-input')
```

Comment:
Find the element you reference and add new attribute called `data-cy`

example: `data-cy = "cohort-slug"`